### ISSUE TITLE
Uploading PDF and/or images files on email blast

### ISSUE THREAD

**Issue by miradoro** [soonick/poMMo#147]
Hi

Is it possible to upload pdfs and/or images to every email campaign?

Please advise

**Comment by soonick** [comment]
Hello @miradoro,

It is possible:

![screenshot](https://f.cloud.github.com/assets/281836/700325/a534cd76-dd43-11e2-8e01-58564da22bba.png)

**Comment by miradoro** [comment]
Hi Soonick

I checked but it seems i do not have that feature activated. I am sending you a screen capture for your review.

![email](https://f.cloud.github.com/assets/367671/701940/d3be7f18-dd82-11e2-9c37-c6eba97860b7.jpg)

**Comment by soonick** [comment]
This is probably because you have an old version of pommo. You can get the newest from https://github.com/soonick/poMMo, look for the Download Zip button.

If you are going to upgrade from an old installation I strongly recommend to make a back up of your database and read https://github.com/soonick/poMMo/wiki/Troubleshooting#unable-to-log-in-after-upgrade if you have problems.

Regard,
Adrian

**Comment by miradoro** [comment]
Hi Adrian

Thanks for your help
I am seeing this error on my browser which i think it is a problem on my server

Forbidden

## You don't have permission to access /prnewsletter on this server.

Apache/2.0.52 (Red Hat) FrontPage/5.0.2.2635 mod_ssl/2.0.52 OpenSSL/0.9.7a Server at www.bfmarketing.com Port 80

**Comment by soonick** [comment]
Hello miradoro,

That could be a lot of things. I would guess you are trying to access yourdomain.com/prnewsletter and the folder prnewsletter doesn't exist or can't be read by the server.

Your system administrator or webmaster should be able to help you with that.

Regards,
Adrian

**Comment by miradoro** [comment]
Hi Adrian
They just told me it was the htaccess filebut now i see the admin page but has no accessis there a default login?

Ivan Torres

Date: Wed, 26 Jun 2013 19:48:33 -0700

**Comment by soonick** [comment]
https://github.com/soonick/poMMo/wiki/Troubleshooting#unable-to-log-in-after-upgrade

**Comment by miradoro** [comment]
Hi adrian

Sdo I run upgrade.sql on php admin?

Sent from Samsung Note 2 T-Mobile 4G LTE Device

-------- Original message --------

**Comment by soonick** [comment]
I recommend backing up you data base first in case something goes wrong.
After you replace the place holders in upgrade.sql you can run it on phpadmin.

**Comment by miradoro** [comment]
Hi

I do not know much about databases
what do you mean by replacing place holders in upgrade.sql ?

Ivan TorresWebmaster and Digital AnalystPhone: 908-342-5526 

This electronic mail transmission is confidential. It is intended only for the named recipient and may contain proprietary or 
legally privileged information. If you have received this transmission in error, please immediately return it to the sender and 
delete original message. Unintended transmission shall not constitute waiver of the attorney-client or any other privilege. 

Esta transmision electronica de correo es confidencial. Es dada solamente para persona o entidad a la que es remitido y 
pueda que contenga informacion legamente priviligeada o apropiada. Si usted recibio esta transmision como error, por favor 
sírvase informarlo de inmediato al remitente y borrar inmediatamente el mensaje original. La transmisión involuntaria no 
constituirá abandono de el cliente-abogado o de ningun otro privilegio.

Date: Fri, 5 Jul 2013 12:13:27 -0700

**Comment by soonick** [comment]
upgrade.sql has instructions to do it:

-- Change all occurrences of <prefix> to the correct prefix your database is using
-- Change all occurrences of <username> with the user you will use to login
-- Change all occurrences of <password> with the password you will use to login

**Comment by miradoro** [comment]
Hi Adrian

My hosting company tried to run the sql file
These are their comments

When I run the script, I get the error:

#1064 - You have an error in your SQL syntax; check the manual that corresponds
to your MySQL server version for the right syntax to use near
'<prefix>_mailings

ADD COLUMN

track tinyint(1) NULL' at line 9 

Ivan TorresWebmaster and Digital AnalystPhone: 908-342-5526 

This electronic mail transmission is confidential. It is intended only for the named recipient and may contain proprietary or 
legally privileged information. If you have received this transmission in error, please immediately return it to the sender and 
delete original message. Unintended transmission shall not constitute waiver of the attorney-client or any other privilege. 

Esta transmision electronica de correo es confidencial. Es dada solamente para persona o entidad a la que es remitido y 
pueda que contenga informacion legamente priviligeada o apropiada. Si usted recibio esta transmision como error, por favor 
sírvase informarlo de inmediato al remitente y borrar inmediatamente el mensaje original. La transmisión involuntaria no 
constituirá abandono de el cliente-abogado o de ningun otro privilegio.

Date: Fri, 5 Jul 2013 14:00:54 -0700

**Comment by soonick** [comment]
Hello @miradoro,

It seems like you didn't replace <prefix> with the prefix your database is using. The default prefix is pommo, so you can try doing a search and replace from <prefix> to pommo. Also don't forget to replace <user> and <password>.

Regards,
Adrian

**Comment by miradoro** [comment]
Hi Adrian
ok i understandSo the prefix i change on the configuration file right?
i didnt undertstand the second partto replace and .
where is 'and .' ?

Ivan TorresWebmaster and Digital AnalystPhone: 908-342-5526 

This electronic mail transmission is confidential. It is intended only for the named recipient and may contain proprietary or 
legally privileged information. If you have received this transmission in error, please immediately return it to the sender and 
delete original message. Unintended transmission shall not constitute waiver of the attorney-client or any other privilege. 

Esta transmision electronica de correo es confidencial. Es dada solamente para persona o entidad a la que es remitido y 
pueda que contenga informacion legamente priviligeada o apropiada. Si usted recibio esta transmision como error, por favor 
sírvase informarlo de inmediato al remitente y borrar inmediatamente el mensaje original. La transmisión involuntaria no 
constituirá abandono de el cliente-abogado o de ningun otro privilegio.

Date: Mon, 8 Jul 2013 19:41:41 -0700

**Comment by soonick** [comment]
Sorry,

I didn't notice that github markdown stripped special characters. You don't change the prefix on the configuration file, you change the prefix in upgrade.sql

You should try doing a search and replace from <prefix> to pommo.
Also replace <username> with the name of the user you currently use to log-in and <password> with the password.

Regards,
Adrian

**Comment by miradoro** [comment]
hi adrian

i updated the sql with the prexi pommo_
and added it a username and password

but it still does not let me log in

Ivan TorresWebmaster and Digital AnalystPhone: 908-342-5526 

This electronic mail transmission is confidential. It is intended only for the named recipient and may contain proprietary or 
legally privileged information. If you have received this transmission in error, please immediately return it to the sender and 
delete original message. Unintended transmission shall not constitute waiver of the attorney-client or any other privilege. 

Esta transmision electronica de correo es confidencial. Es dada solamente para persona o entidad a la que es remitido y 
pueda que contenga informacion legamente priviligeada o apropiada. Si usted recibio esta transmision como error, por favor 
sírvase informarlo de inmediato al remitente y borrar inmediatamente el mensaje original. La transmisión involuntaria no 
constituirá abandono de el cliente-abogado o de ningun otro privilegio.

Date: Tue, 9 Jul 2013 20:56:23 -0700

**Comment by miradoro** [comment]
hi

i just noticed i get this message on IE

Warning: filesize() [function.filesize]: stat failed for /vservers/locura/htdocs/prnewsletter/cache/ERROR_LOG in /vservers/locura/htdocs/prnewsletter/classes/Pommo_Helper_Maintenance.php on line 31

Warning: Cannot modify header information - headers already sent by (output started at /vservers/locura/htdocs/prnewsletter/classes/Pommo_Helper_Maintenance.php:31) in /vservers/locura/htdocs/prnewsletter/classes/Pommo.php on line 475

Ivan TorresWebmaster and Digital AnalystPhone: 908-342-5526 

This electronic mail transmission is confidential. It is intended only for the named recipient and may contain proprietary or 
legally privileged information. If you have received this transmission in error, please immediately return it to the sender and 
delete original message. Unintended transmission shall not constitute waiver of the attorney-client or any other privilege. 

Esta transmision electronica de correo es confidencial. Es dada solamente para persona o entidad a la que es remitido y 
pueda que contenga informacion legamente priviligeada o apropiada. Si usted recibio esta transmision como error, por favor 
sírvase informarlo de inmediato al remitente y borrar inmediatamente el mensaje original. La transmisión involuntaria no 
constituirá abandono de el cliente-abogado o de ningun otro privilegio.

Date: Tue, 9 Jul 2013 20:56:23 -0700

**Comment by miradoro** [comment]
Hi Adrian

I got it !!!

I had the port defined on the configuration file and that was throwing the error

It seems like i have it up and running but so far when i go to admin page->users->configure

i get the word 'processing...' 

the information on the other tabs work just fine

any ideas?

Ivan TorresWebmaster and Digital AnalystPhone: 908-342-5526 

This electronic mail transmission is confidential. It is intended only for the named recipient and may contain proprietary or 
legally privileged information. If you have received this transmission in error, please immediately return it to the sender and 
delete original message. Unintended transmission shall not constitute waiver of the attorney-client or any other privilege. 

Esta transmision electronica de correo es confidencial. Es dada solamente para persona o entidad a la que es remitido y 
pueda que contenga informacion legamente priviligeada o apropiada. Si usted recibio esta transmision como error, por favor 
sírvase informarlo de inmediato al remitente y borrar inmediatamente el mensaje original. La transmisión involuntaria no 
constituirá abandono de el cliente-abogado o de ningun otro privilegio.

Date: Tue, 9 Jul 2013 20:56:23 -0700

**Comment by soonick** [comment]
Can you please attach a screenshot?

**Comment by miradoro** [comment]
Hi Adrian

I have a couple of errors:
When i click on mailing, i get this

A Mailing is currently processing. Visit the Status page to check its 
progress.See notices.jpg

Even if i click cancel, it still process the mailing

---

On setup-> configure 
i get this processing text...it does not load data

---

Also i get attachment failed when attaching files 
I could not send u a screen capture because the mailing processing does not go away

Ivan TorresWebmaster and Digital AnalystPhone: 908-342-5526 

This electronic mail transmission is confidential. It is intended only for the named recipient and may contain proprietary or 
legally privileged information. If you have received this transmission in error, please immediately return it to the sender and 
delete original message. Unintended transmission shall not constitute waiver of the attorney-client or any other privilege. 

Esta transmision electronica de correo es confidencial. Es dada solamente para persona o entidad a la que es remitido y 
pueda que contenga informacion legamente priviligeada o apropiada. Si usted recibio esta transmision como error, por favor 
sírvase informarlo de inmediato al remitente y borrar inmediatamente el mensaje original. La transmisión involuntaria no 
constituirá abandono de el cliente-abogado o de ningun otro privilegio.

Date: Fri, 12 Jul 2013 23:02:08 -0700

**Comment by soonick** [comment]
Can you please open one ticket for each of the new issues you are having. This doesn't seem to be related to problems uploading PDFs anymore.

Also, Can you please remove your signature from future messages? it makes the conversation hard to follow.

Thanks,
Adrian
